.github/workflows/ci.yml

```diff
@@ -209,31 +209,6 @@ jobs:
           cl.exe /DJS_NAN_BOXING=0 /Zs cxxtest.cc
           cl.exe /DJS_NAN_BOXING=1 /Zs cxxtest.cc
 
-  windows-msvc-vs2019:
```

Comment:
Can you mention the requirement of atomics in https://quickjs-ng.github.io/quickjs/supported_platforms ?